WooCommerce/Classes/ViewRelated/Orders/Order Filters/FilterOrderListViewModel.swift

```diff
@@ -322,3 +369,20 @@ extension CustomerFilter: FilterType {
     /// Whether the filter is set to a non-empty value.
     var isActive: Bool { true }
 }
+
+extension FilterOrderListViewModel {
+    enum SalesChannelFilter: FilterType {
+        case pointOfSale
+
+        var description: String {
+            switch self {
+            case .pointOfSale:
+                return "pos"
```

Comment:
Should this be upper case?

Reply:
It can! And most likely it will soon, updated https://github.com/woocommerce/woocommerce-ios/pull/15878/commits/de53799ff5b838f4587df13bee01ad82576d44c4